### PR TITLE
Task nxp 26961 drive polymer 3

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: addons/nuxeo-drive/elements/nuxeo-authentication-tokens-management.js [hunk 1]
```diff
@@ -80,17 +82,17 @@
       }
     </style>
 
-    <nuxeo-resource auto id="tokens" path="/token" params="[[_params(application)]]" on-response="_handleTokens"></nuxeo-resource>
+    <nuxeo-resource auto="" id="tokens" path="/token" params="[[_params(application)]]" on-response="_handleTokens"></nuxeo-resource>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-authentication-tokens-management.js]:
```suggestion
    <nuxeo-resource auto id="tokens" path="/token" params="[[_params(application)]]" on-response="_handleTokens"></nuxeo-resource>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-desktop-packages.js [hunk 2]
```diff
@@ -92,62 +94,58 @@
         <div class="row">
           <div class="cell flex"><span class="platform">[[pkg.platform]]</span></div>
           <div class="cell flex-3">
-            <a href$="[[pkg.url]]" tabindex="-1" target="_blank">
-              <paper-button noink>
+            <a href\$="[[pkg.url]]" tabindex="-1" target="_blank">
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-desktop-packages.js]:
```suggestion
            <a href$="[[pkg.url]]" tabindex="-1" target="_blank">
```

File: addons/nuxeo-drive/elements/nuxeo-drive-desktop-packages.js [hunk 3]
```diff
@@ -92,62 +94,58 @@
...
+              <paper-button noink="">
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-desktop-packages.js]:
```suggestion
              <paper-button noink>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+/**
+`nuxeo-drive-edit-button`
+@group Nuxeo UI
+@element nuxeo-drive-edit-button
+*/
+Polymer({
+  _template: html`
+    <style include="nuxeo-action-button-styles"></style>
+
+    <nuxeo-resource id="token" path="/token" params="{&quot;application&quot;: &quot;Nuxeo Drive&quot;}"></nuxeo-resource>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js]: inline JSON is screwed up?
```suggestion
    <nuxeo-resource id="token" path="/token" params='{"application": "Nuxeo Drive"}'></nuxeo-resource>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+    <template is="dom-if" if="[[_isAvailable(document,blob)]]">
+      <div class="action" on-tap="_go">
+        <paper-icon-button noink="" icon="icons:open-in-new" id="driveBtn"></paper-icon-button>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js]:
```suggestion
        <paper-icon-button noink icon="icons:open-in-new" id="driveBtn"></paper-icon-button>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        <span class="label" hidden\$="[[!showLabel]]">[[i18n('driveEditButton.tooltip')]]</span>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js]:
```suggestion
        <span class="label" hidden$="[[!showLabel]]">[[i18n('driveEditButton.tooltip')]]</span>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-sync-roots-management.js [hunk 7]
```diff
@@ -76,8 +78,8 @@
       }
     </style>
 
-    <nuxeo-operation auto id="roots" op="NuxeoDrive.GetRoots" on-response="_handleRoots"></nuxeo-operation>
-    <nuxeo-operation id="disable" op="NuxeoDrive.SetSynchronization" params='{"enable": false}'></nuxeo-operation>
+    <nuxeo-operation auto="" id="roots" op="NuxeoDrive.GetRoots" on-response="_handleRoots"></nuxeo-operation>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-sync-roots-management.js]:
```suggestion
    <nuxeo-operation auto id="roots" op="NuxeoDrive.GetRoots" on-response="_handleRoots"></nuxeo-operation>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-sync-roots-management.js [hunk 8]
```diff
@@ -76,8 +78,8 @@
...
+    <nuxeo-operation id="disable" op="NuxeoDrive.SetSynchronization" params="{&quot;enable&quot;: false}"></nuxeo-operation>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-sync-roots-management.js]:
```suggestion
    <nuxeo-operation id="disable" op="NuxeoDrive.SetSynchronization" params='{"enable": false}'></nuxeo-operation>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-sync-toggle-button.js [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+      ::slotted(iron-icon:hover) {
+        fill: var(--nuxeo-link-hover-color);
+      }
+
+      :host([synchronized]) ::slotted(iron-icon) {
+        fill: var(--icon-toggle-pressed-color, --nuxeo-action-color-activated);
+      }
+    </style>
+
+    <nuxeo-operation auto="" op="NuxeoDrive.GetRoots" on-response="_handleRoots"></nuxeo-operation>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-sync-toggle-button.js]:
```suggestion
    <nuxeo-operation auto op="NuxeoDrive.GetRoots" on-response="_handleRoots"></nuxeo-operation>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-sync-toggle-button.js [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+    <nuxeo-operation id="op" op="NuxeoDrive.SetSynchronization" input="[[document.uid]]"></nuxeo-operation>
+
+    <template is="dom-if" if="[[_isAvailable(document, synchronizationRoot)]]">
+      <div class="action" on-tap="toggle">
+        <paper-icon-button id="syncBut" icon="[[_icon(synchronized)]]"></paper-icon-button>
+        <span class="label" hidden\$="[[!showLabel]]">[[_label]]</span>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-sync-toggle-button.js]:
```suggestion
        <span class="label" hidden$="[[!showLabel]]">[[_label]]</span>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+      </div>
+      <paper-tooltip for="driveBtn">[[i18n('driveEditButton.tooltip')]]</paper-tooltip>
+    </template>
+
+    <nuxeo-dialog id="dialog" with-backdrop="">
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js]:
```suggestion
    <nuxeo-dialog id="dialog" with-backdrop>
```

File: addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+      <div class="vertical layout">
+        <h1>[[i18n('driveEditButton.dialog.heading')]]</h1>
+        <nuxeo-drive-desktop-packages></nuxeo-drive-desktop-packages>
+      </div>
+      <div class="buttons">
+        <paper-button dialog-dismiss="">[[i18n('command.close')]]</paper-button>
```
Comment (Gabez0r) [inline addons/nuxeo-drive/elements/nuxeo-drive-edit-button.js]:
```suggestion
        <paper-button dialog-dismiss>[[i18n('command.close')]]</paper-button>
```

**Comment by Gabez0r** [comment]
:+1:

**Comment by nelsonsilva** [comment]
@Gabez0r thanks for the review. These are artifacts from the automatic Polymer 3 migration. I'd rather clean it up in the whole codebase in one go once we've migrated everything.

**Pull request merged**